spatialscaper/core.py

```diff
@@ -570,14 +573,17 @@ def get_room_irs_wav_xyz(self, wav=True, pos=True):
             tuple: A tuple containing the impulse responses, their sampling rate, and their XYZ positions.
         """
         if self.format == "foa" and self.room == "metu":
-            raise ValueError(
-                '"metu" room is currently only supported in mic (tetrahedral) format. please check again soon.'
+            room_sofa_path = os.path.join(
```

Comment:
Instead of hardcoding metu em32, perhaps there is a way to express it more generically as a em32 -> mic translation? I guess it depends on how standard the mic indexing is (do eigenmics follow consistent indexing `[5, 9, 25, 21]`? Could this be applied to any "em32" format sofa?)

To avoid unnecessary complexity, maybe it should stay like this for now, but format mapping might be nice to generalize at some point.

Reply:
`prepare_rirs.py` will process all microphone arrays to be either tetrahedral ('mic') or 'foa' for now. The `[5, 9, 25, 21]` indexing in `core.py` has been eliminated. 

In the future we might want to support 32 channels, but for this could only be possible with the METU and ARNI rooms. Let's reconsider this when more rooms are added to the code base.